src/components/editor/PropertyTemplate.jsx

```diff
@@ -16,6 +17,16 @@ class PropertyTemplate extends Component {
           <li>mandatory: <strong>{this.props.propertyTemplates[0].mandatory}</strong></li>
           <li>repeatable: <strong>{this.props.propertyTemplates[0].repeatable}</strong></li>
           <li>contained resourceTemplates[] length: <strong>{this.props.propertyTemplates[0].resourceTemplates.length}</strong></li>
+          <li>
+            <ValueConstraints
+              valueTemplateRefs = {this.props.propertyTemplates[0].valueConstraint.valueTemplateRefs}
+              useValuesFrom = {this.props.propertyTemplates[0].valueConstraint.useValuesFrom}
+              valueDataType = {this.props.propertyTemplates[0].valueConstraint.valueDataType}
+              repeatable = {this.props.propertyTemplates[0].valueConstraint.repeatable}
+              editable = {this.props.propertyTemplates[0].valueConstraint.editable}
+              defaults = {this.props.propertyTemplates[0].valueConstraint.defaults}
```

Comment:
this seems like a prime case for using the spread operator:  https://zhenyong.github.io/react/docs/jsx-spread.html  or  https://www.robinwieruch.de/react-pass-props-to-component/#react-props-syntax-spread-rest   

"The props spreading can be used to spread a whole object with key value pairs down to a child component. It has the same effect as passing each value of the object by its own to the component as before. The child component gets the props the same way as before too."

If we're eager to get this PR in, make a follow-on ticket for this?  And refer to that ticket in the code?

Reply:
👌

Reply:
There was no need to send all of them. Now I am just passing `{...this.props.propertyTemplates[0].valueConstraint}`